test/e2e_node/podresources_test.go

```diff
@@ -1027,8 +1027,8 @@ var _ = SIGDescribe("POD Resources", framework.WithSerial(), feature.PodResource
 					}
 					pod := makePodResourcesTestPod(pd)
 					pod.Spec.Containers[0].Command = []string{"sh", "-c", "/bin/true"}
-					pod = e2epod.NewPodClient(f).CreateSync(ctx, pod)
-					defer e2epod.NewPodClient(f).DeleteSync(ctx, pod.Name, metav1.DeleteOptions{}, time.Minute)
+					pod = e2epod.NewPodClient(f).Create(ctx, pod)
```

Comment:
this is fine for the reasons you explained - and we do WaitForPodCondition shortly after anyway